playbooks/infra/deploy-bm-hypervisor.yml

```diff
@@ -199,10 +199,10 @@
         autoconnect: true
 
     - name: Reload NetworkManager connections
-      ansible.builtin.shell: |
-        "nmcli con down  {{ (net_config | from_yaml)['interface_name'] }} &&
-         nmcli con up  {{ (net_config | from_yaml)['interface_name'] }} &&
-         nmcli con up bridge-baremetal && nmcli con up  {{ (net_config | from_yaml)['interface_name'] }}"
+      ansible.builtin.shell: >
+        nmcli con down  {{ (net_config | from_yaml)['interface_name'] }} &&
+          nmcli con up  {{ (net_config | from_yaml)['interface_name'] }} &&
+          nmcli con up bridge-baremetal && nmcli con up  {{ (net_config | from_yaml)['interface_name'] }}
```

Comment:
` nmcli con up  {{ (net_config | from_yaml)['interface_name'] }}` looks like this part is already happening on line 204.

Reply:
And maybe this line:

`nmcli con up bridge-baremetal && nmcli con up  {{ (net_config | from_yaml)['interface_name'] }}`

should be split into two:

```
nmcli con up bridge-baremetal &&
nmcli con up  {{ (net_config | from_yaml)['interface_name'] }}
```

Reply:
@ccardenosa Thanks lines are formatted.

Reply:
@shaior No, we need to bring this interface up 2 times due to some **weird bridge interface behavior.** It helps bring bridge up